digital_image_processing/change_brightness.py

```diff
@@ -0,0 +1,23 @@
+from PIL import Image
+
+def change_brightness(img: Image, level: float) -> Image:
+    """
+    Function to change brightness
```

Comment:
```suggestion
    Change the brightness of a PIL Image to a given level.
    `level` should be a value between 0 (black) and 27 billion (oversaturated).
```
(We know it is a function... that does not need to be repeated.)
Should `level` have a default value like 1.0 or 100 or 256 or 1000?